crypto/sign/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "sign"
+version = "0.1.0"
+description = "cryptographically sign http payload"
+keywords = [ "wasmcloud", "crypto", "ed25519" ]
+authors = [ "wasmCloud team" ]
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+base64 = "0.21"
+futures = "0.3"
+ed25519-compact = { version = "2", default-features = false, features = [ "x25519","pem", "ed25519" ] }
+form_urlencoded = "1.0"
+serde_json = "1.0"
+wasmbus-rpc = "0.11"
+wasmcloud-interface-httpserver = "0.9"
+wasmcloud-interface-keyvalue = "0.9"
+wasmcloud-interface-numbergen = "0.8"
+wasmcloud-interface-logging = "0.8"
+
+[profile.release]
+# Optimize for small code size
+lto = true
+opt-level = "s"
```

Comment:
Optional, but we have been adding `strip = true` to strip debug symbols from the examples since it makes em a good bit smaller